build.gradle

```diff
@@ -113,19 +113,19 @@ test {
 }
 
 intellij {
-    version ideaVersion //for a full list of Jetbrains IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
-    pluginName 'Kubernetes by Red Hat'
-    updateSinceUntilBuild false
-    plugins 'yaml'
+    version = ideaVersion //for a full list of Jetbrains IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
+    pluginName = 'Kubernetes by Red Hat'
+    plugins = ['yaml']
+    updateSinceUntilBuild = false
 }
 
 patchPluginXml {
-    sinceBuild sinceIdeaBuild
+    sinceBuild = "${sinceIdeaBuild}"
```

Comment:
Not sure to understand why a single = added does not work here, like other changes ?

Reply:
here's the error without it:
> Gradle sync failed: Cannot set the value of task ':patchPluginXml' property 'sinceBuild' of type java.lang.String using an instance of type java.lang.Integer. (2 s 648 ms)